apps/gallery/js/ImageEditor.js

```diff
@@ -223,6 +223,13 @@ var exposureSlider = (function() {
   thumb.addEventListener('touchend', function(e) {
     thumb.classList.remove('active');
   });
+  slider.addEventListener('keypress', function(e) {
+    if (e.keyCode == KeyEvent.DOM_VK_DOWN) {
+      setExposure(currentExposure - 0.25);
+    } else {
+      setExposure(currentExposure + 0.25);
+    }
+  });
```

Comment:
Is it possible to test the above code on flame-kk? I tried with screen reader enabled in settings that failed to get exposure slider to work.